specification/resources/kubernetes/models/cluster.yml

```diff
@@ -71,8 +73,11 @@ properties:
       - k8s:bd5f5959-5e1e-4205-a714-a914373942af
       - production
       - web-team
-    description: An array of tags applied to the Kubernetes cluster. All
+    description: An array of tags to apply to the Kubernetes cluster. All
       clusters are automatically tagged `k8s` and `k8s:$K8S_CLUSTER_ID`.
+      <br><br>Requires `tag:read` and `tag:create` scope.
```

Comment:
```suggestion
      <br><br>Requires `tag:read` and `tag:create` scope, as well as `tag:delete` if existing tags are getting removed,
```

Reply:
Or maybe this needs a separate model - for POST only tag:read and tag:create are necessary, whereas for PUT it's tag:read, tag:create, and tag:delete